Kickstarter-iOS/Features/PledgePaymentMethods/Controller/PledgePaymentMethodsViewControllerTests.swift

```diff
@@ -51,9 +51,7 @@ final class PledgePaymentMethodsViewControllerTests: TestCase {
           checkoutId: "checkoutId",
           reward: reward,
           context: .pledge,
-          refTag: nil,
-          pledgeTotal: Double.nan,
-          paymentSheetType: .setupIntent
```

Comment:
n.B. that this stuff doesn't need to be passed around any more, because the payment sheet no longer needs a total or a type.